README.md

```diff
@@ -1,56 +1,90 @@
+you will want to be able to start tracking your changes and storing these changes 
+somewhere accessible so that other developers can review or collaborate on the project--
+or just put it somewhere you can come back to it later, regardless of what may happen 
+to the physical files on your computer. This is where `git` comes in! `Git` will allow 
+the users to keep track of these changes and keep detailed tabs and backups of your project.
 
-`git init` initializes a new Git repository in your current directory. Type `git init` from within your `next-big-thing` project directory.
+Now that we've established what `git` does and how useful it can be, how 
+can we start using it? I you haven't already generated some files on your local machine at 
+this point--You can do so by using these commands to create a project:
 
 ```
-next-big-thing $ git init
-Initialized empty Git repository in /Users/avi/next-big-thing/.git/
+~ $ mkdir my-git-project //Creates new directory
+~ $ cd my-git-project //Moves into the newly created directory
 ```
 
-After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/next-big-thing/.git/`. Git is letting you know that it created a new repository within the hidden `.git` folder in `next-big-thing`. This hidden directory, `.git`, is what Git uses to keep track of all the different versions of your code.
+The files are available now, so how do we get them in the cloud so we can collaborate with 
+other developers or backup your progress thus far? 
+
+## State initializing Git repository with `git init`
+
+Make sure you are in the directory where you want to initiliaze `git` on your computer. If 
+you would like control, you can use a few commands to get things up and running quickly. In 
+your terminal type:
+
+```
+my-git-project $ git init
+Initialized empty Git repository in /Users/avi/my-git-project/.git/
+```
 
-You don't need to do anything with the `.git` directory –– it'll remain hidden. Just remember that if a directory has a `.git` directory inside of it, it is controlled by Git. It's also nice to know there's no magic behind the process: if Git is keeping track of multiple versions of files that you can't see, Git has to be putting them somewhere inside your computer. That somewhere is `.git`.
+After entering `git init`, the output in your terminal reads `Initialized empty Git repository in /your/path/here/my-git-project/.git/`. Git is letting you know that it created a new repository 
+within the hidden `.git` folder in `my-git-project`. This hidden directory, `.git`, is what Git uses 
+to keep track of all the different versions of your code.
 
-At this point, though, there's nothing for Git to keep track of. We'll have to create some files and folders in our project first.
+You won't need to do anything with the `.git` directory –– it'll remain hidden. Just remember that if 
```

Comment:
Is your wrap-length something odd here? The style guide urges wrap at 80 columns and I'll tolerate nowrap whatsoever (although it makes GitHub reviews harder).